src/main/kotlin/org/pkl/intellij/psi/Extensions.kt

```diff
@@ -413,7 +413,7 @@ fun resolveModuleUriGlob(element: PklModuleUri, context: PklProject?): List<PklM
   element.escapedContent
     ?.let { text ->
       val psiManager = PsiManager.getInstance(element.project)
-      CachedValuesManager.getManager(element.project).getCachedValue(element) {
+      element.getContextualCachedValue(context) {
```

Comment:
Unrelated: this was causing warning logs because IJ doesn't like the context being used as a cache value dependency.